### PR TITLE
Feature/selective process

### PR DESCRIPTION
**Pull request by alanaraujop**
## Description

Foi criada a seção com as etapas do processo seletivo.  Conforme a data vai avançando a etapa seguinte ficará marcada de verde, recebendo destaque.

### Steps

1. Navegue até a seção 'Processo seletivo' e observe se a etapa seguinte está verde
2. Modifique a data do relogio do computador para uma superior a fase atual
3. Feche o browser e abra novamente
4. Verifique se a nova etapa recebeu destaque como esperado

## Screenshots / Videos

image

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

File: src/components/SelectiveProcess/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+export type Question = {
+  question: string;
+  answer: string;
+  key: string;
+};
+
+export const SelectiveProcess = () => (
+  <S.Section>
+    <S.Title variant="h2">
+      Sobre o <br /> <span>processo seletivo </span>
```
Comment (jonataspinto) [inline src/components/SelectiveProcess/index.tsx:13]: Bom, acho que você não acompanhou o que foi rolando, mas decidimos usar o `dangernousInnerHtml` para adicionar os textos. Pois estamos estudando a viabilidade de CMS com o admin que temos hoje. A página de peguntas-frequentes já está pegando de lá inclusive.

File: src/components/SelectiveProcess/styles.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  }
+  @media (max-width: ${({ theme: { media } }) => media.tablet}) {
+    height: 4rem;
+    font-size: ${({ theme }) => theme.typography.body1.fontSize};
+  }
+`;
+
+export const ScheduleWrapper = styled.div`
+  display: flex;
+  margin-top: 70px;
```
Comment (jonataspinto) [inline src/components/SelectiveProcess/styles.ts:60]: rem

File: src/components/SelectiveProcess/styles.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+`;
+
+export const ScheduleContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  grid-gap: 48px;
```
Comment (jonataspinto) [inline src/components/SelectiveProcess/styles.ts:66]: rem

File: src/components/SelectiveProcess/index.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import { Logo, Typography } from '..';
+import * as S from './styles';
+import { scheduleMock } from '../../mocks/Schedule.mock';
+export type Question = {
```
Comment (jonataspinto) [inline src/components/SelectiveProcess/index.tsx:4]: Está usando?

**Review by jonataspinto — CHANGES_REQUESTED** [review]

File: src/components/SelectiveProcess/index.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      Tecnologia da Informação. Por isso, oferecemos uma formação objetiva e
+      focada nas exigências do mercado de TI. Os dois cursos incluem ainda
+      conhecimentos em métodos de gestão ágil de projetos utilizados no mercado
+      para o desenvolvimento de trabalhos em equipe, além de assessoria no
+      Linkedin e direcionamento de carreira.
+    </S.Text>
+    <S.ScheduleWrapper>
+      <S.ScheduleDotsWrapper>
+        {scheduleMock.map((schedule, index) => {
+          const isTheLast = index === scheduleMock.length - 1;
```
Comment (ramonxm) [inline src/components/SelectiveProcess/index.tsx:26]: Uma outra forma de acessar o último elemento do array é  utilizar o `at`

Exemplo:  `array.at(-1)`

Comment (jonataspinto) [reply src/components/SelectiveProcess/index.tsx]: rolou não.. deu chabú

Comment (ramonxm) [reply src/components/SelectiveProcess/index.tsx]: Ah, eu botei só como comentário, devia ter especificado, o at retorna o value do array nessa posição, não o número. No caso ele compara se o index é o ultimo ali, ai não se aplica o at nesse caso

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.6 | ✅ |
| First Contentful Paint | 3.3 s |  |
| Time to Interactive | 4.9 s |  |
| Speed Index | 3.8 s |  |
| Total Blocking Time | 580 ms |  |
| Largest Contentful Paint | 4.8 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.85 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 6.3 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 370 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.54 | ✅ |
| First Contentful Paint | 3.4 s |  |
| Time to Interactive | 4.8 s |  |
| Speed Index | 3.7 s |  |
| Total Blocking Time | 710 ms |  |
| Largest Contentful Paint | 5.5 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.91 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 5.5 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 190 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.54 | ✅ |
| First Contentful Paint | 3.3 s |  |
| Time to Interactive | 4.9 s |  |
| Speed Index | 4.0 s |  |
| Total Blocking Time | 650 ms |  |
| Largest Contentful Paint | 5.8 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.71 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 7.1 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 820 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by ramonxm** [comment]
@jonataspinto vai subir mais alguma coisa?

**Comment by jonataspinto** [comment]
> @jonataspinto vai subir mais alguma coisa?

Mergeando

**Review by jonataspinto — APPROVED** [review]

**Pull request merged**